schema-engine/connectors/sql-schema-connector/src/warnings/generators.rs

```diff
@@ -332,6 +344,34 @@ pub(crate) struct IndexedColumn {
     pub(crate) column_name: String,
 }
 
+/// A check constraint that triggered a warning.
+#[derive(Serialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct CheckConstraint {
+    /// The name of the namespace
+    // pub(crate) namespace: String,
+    /// The name of the table
+    // pub(crate) table: String,
+    /// The name of the constraint
+    pub(crate) name: String,
+    /// The definition of the constraint
+    pub(crate) definition: String,
+}
+
+/// An exclusion constraint that triggered a warning.
+#[derive(Serialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct ExclusionConstraint {
+    /// The name of the namespace
+    /// pub(crate) namespace: String,
+    /// The name of the table
+    /// pub(crate) table: String,
```

Comment:
Should that commented out code be there? Also, since it's all doc comments, it's going to show up in the rustdoc output for the `name` field.

Reply:
(same in CheckConstraint right above)